app/views/anonymous_feedback/_feedback.html.erb

```diff
@@ -15,6 +15,11 @@
     <br/>
     <strong>comment</strong>: <%= anonymous_contact.details %>
   <% end %>
+<% elsif anonymous_contact.type == "service-feedback-rating" %>
```

Comment:
No, sorry that was a version control issue that also impacted how the service feedback aggregate details were called, but amended now. Tweaked the wording too following Liz's feedback, cf. updated screenshot in https://github.com/alphagov/support-api/pull/82 .